sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncReadBenchmark.java

```diff
@@ -53,7 +54,9 @@ protected void performWorkload(BaseSubscriber<CosmosAsyncItemResponse> baseSubsc
         PojoizedJson doc = docsToRead.get(index);
 
         String partitionKeyValue = doc.getId();
-        Mono<CosmosAsyncItemResponse> result = cosmosAsyncContainer.getItem(doc.getId(), partitionKeyValue).read();
+        Mono<CosmosAsyncItemResponse<PojoizedJson>> result = cosmosAsyncContainer.readItem(doc.getId(),
+                                                                                           new PartitionKey(partitionKeyValue),
```

Comment:
can't we assume that the second param is always partition key?

that way the caller doesn't have to explicitly wrap pk value in PartitionKey and implementation will take care of that.

thoughts?

Reply:
We are following the .net model of making partitionkey object manadatory. 
https://github.com/Azure/azure-cosmos-dotnet-v3/blob/8f1375a30799b3acf95d843b2db0c2447cbc3876/Microsoft.Azure.Cosmos/src/Resource/Container/Container.cs#L476